### ISSUE TITLE
Can we use SIXray DataSet in Object Detection?

### ISSUE THREAD

**Issue by tianfengyijiu** [MeioJane/SIXray#2]
Hi,
Thanks very much for your jobs.
I have read your paper about SIXray so I want to use this Dataset in Object Detection such as Faster-Rcnn or YOLOV3.
Now, I want to get all annotations about the prohibited item, can you support me?

**Comment by GokulNC** [comment]
Yes, the dataset contains annotations for around 1400 images which you can use for detection. I'm not sure about the train-test split used by the authors for the localization task mentioned in the paper.

Can you please help us here? @MeioJane

**Comment by MeioJane** [comment]
> Hi,
> Thanks very much for your jobs.
> I have read your paper about SIXray so I want to use this Dataset in Object Detection such as Faster-Rcnn or YOLOV3.
> Now, I want to get all annotations about the prohibited item, can you support me?

I have update all the annotations

**Comment by jodumagpi** [comment]
This should be closed because the complete bounding box annotations are already posted
